packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -382,10 +338,38 @@ export async function getBuilder(
       }
       throw err;
     }
+
+    // If it's a bundled builder, then cache the require call
+    if (bundledBuilder) {
+      localBuilders[builderPkg] = builderWithPkg;
+    }
   }
   return builderWithPkg;
 }
 
+export function isBundledBuilder(
+  parsed: npa.Result,
+  distTag: string,
+  pkg: PackageJson
+): boolean {
+  if (!parsed.name || !pkg.dependencies) {
+    return false;
+  }
+
+  const bundledVersion = pkg.dependencies[parsed.name];
+  if (bundledVersion) {
+    if (distTag !== 'canary' && !bundledVersion.includes('canary')) {
```

Comment:
I'm a little confused here. Why is canary relevant to determine if the builder is bundled?

Reply:
I would expect something like:

```js
const bundledVersion = pkg.dependencies[parsed.name];
return bundledVersion && (!parsed.version || bundledVersion === parsed.version);
```

Reply:
The idea is that if the now.json specifies `@now/node` (stable), but Now CLI has a canary version of `@now/node` installed, then `now dev` will install the stable version to the builders cache directory and use the stable builder (and vice-versa).

Reply:
But what about if someone has `@now/node@1.0.0-canary.1` in their `now.json`?

I think your code will return true that this is bundled even though it is not.

Reply:
Oh ya, good point. I'll add a check if it's an explicit version.